internal_api.go

```diff
@@ -51,11 +61,18 @@ func (e *Enforcer) addPolicy(sec string, ptype string, rule []string) (bool, err
 // addPolicies adds rules to the current policy.
 // removePolicies removes rules from the current policy.
 func (e *Enforcer) addPolicies(sec string, ptype string, rules [][]string) (bool, error) {
-	rulesAdded := e.model.AddPolicies(sec, ptype, rules)
+	rulesAdded, effects := e.model.AddPolicies(sec, ptype, rules)
```

Comment:
can `effects` be `nil` here when there is a dup

Reply:
Returns `nil` will convert zero slice, it's working.